applications/MeshingApplication/tests/mmg_lagrangian_test/beam2D_test_parameters.json

```diff
@@ -138,8 +138,12 @@
         "help"                  : "This process compares two files and executes an Unittest Assert",
         "process_name"          : "CompareTwoFilesCheckProcess",
         "Parameters"            : {
-            "file_name_1"       : "mmg_lagrangian_test/beam2D_test_step=10.o.mesh",
```

Comment:
this is a perfect case for a test written without using the standard main. That would also be useful for understanding how to use it.